cli/src/cluster_query.rs

```diff
@@ -166,14 +166,7 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .default_value("15")
                         .help("Wait up to timeout seconds for transaction confirmation"),
                 )
-                .arg(
-                    Arg::with_name(COMMITMENT_ARG.name)
-                        .long(COMMITMENT_ARG.long)
-                        .takes_value(true)
-                        .possible_values(&["default", "max", "recent", "root"])
-                        .value_name("COMMITMENT_LEVEL")
-                        .help("Wait until the transaction is confirmed at selected commitment level"),
```

Comment:
I left this explicitly for the help message, which seemed more clear for `ping`. Not worth the duplication?

Reply:
IMO no :)